### ISSUE TITLE
Holding keys to change volume only changes it once, and cannot scroll to change volume in some screens

### ISSUE THREAD

**Issue by Lemmmy** [ppy/osu#31267]
### Type

Game behaviour

### Bug description

As of the latest lazer update, there's a few very noticable changes to volume control behaviour. I'm not sure if any of these are intentional, or a regression caused by #31146 or #31147?

- ~~In the main menu and gameplay, holding down the keys I have bound for volume either don't change the volume at all, or change it by only one increment if the volume controls are already visible. Alt+scroll still works as expected here (and in song select, too). Holding down a key worked in previous versions.~~ Fixed by #31282.
- In several screens, alt+scroll to change the volume no longer works at all. This worked in the previous version. As far as I can tell, there's no way to change the volume in these screens anymore? Screens not working:
  - Daily challenge (this is the only one I'm certain worked before)
  - Song results
  - Multi list
  - Multi room
  - Playlist list
  - Playlist room
  - Beatmap editor (keybinds do work here)
  - Skin editor (keybinds do work here)

### Screenshots or videos

https://github.com/user-attachments/assets/d4192c49-a230-422f-8668-fa8e0765d5c5

Apologies for crude video, didn't have any kind of input overlays on hand

My keybindings:

![osu!_77ocNY7vXB](https://github.com/user-attachments/assets/a2fa0015-cfac-4deb-a738-1ce317ead00c)

### Version

2024.1224.1-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/18242288/compressed-logs.zip)

**Comment by peppy** [comment]
To fix alt-scroll, I believe we should add a second global binding for this (and not do explicit implementation again as it was before).

For the other cases it sounds like a presence issue.

**Comment by Lemmmy** [comment]
Thanks for fixing both issues!

> You can still use global bindings to adjust volume. You can also use the toolbar.

It probably doesn't help that my global bindings are just letter keys, so they don't seem to work in any screen with a textbox (unless there's a way to unfocus the textbox I'm probably missing?) 

The toolbar is great to know though—thanks for that.

**Comment by TheEZIC** [comment]
The same issues applies to score result screen

**Comment by GGchung69** [comment]
got a new update but scroll is still broken

**Comment by peppy** [comment]
> got a new update but scroll is still broken

what issue are you seeing?

**Comment by GGchung69** [comment]
same as the author

**Comment by Lemmmy** [comment]
> same as the author 

The fix hasn't been merged (so isn't in the game) yet: https://github.com/ppy/osu/pull/31281

**Comment by GGchung69** [comment]
oh

**Comment by peppy** [comment]
I don't believe fixing the remaining portion of this issue is in our best interest.

For users which are not happy with this, here's some alternatives:

- You can bind alt-scroll in settings to your volume adjust – it will work everywhere again (but it will override some things like editor distance adjust).
- You can scroll over the volume icon on the toolbar to adjust volume even without holding alt:

![Image](https://github.com/user-attachments/assets/2b3c450b-5873-4d4f-95c3-dad874dd7024)

Going to close this one out for now.
